python/paddle/fluid/tests/unittests/test_dygraph_to_static_basic_api_transformation.py

```diff
@@ -28,19 +28,25 @@
 
 
 def dyfunc_to_variable(x):
-    res = fluid.dygraph.to_variable(x)
+    res = fluid.dygraph.to_variable(x, name=None, zero_copy=None)
+    return res
```

Comment:
If you should push a commit to fix conflict, could you move this unittest file to dir `dygraph_to_static`?

Reply:
OK. And other test files about dygraph_to_static will be moved.